dev-lang/rust/rust-1.27.0.recipe

```diff
@@ -0,0 +1,235 @@
+SUMMARY="Modern and safe systems programming language"
+DESCRIPTION="Rust is a systems programming language that runs blazingly fast, \
+prevents almost all crashes*, and eliminates data races."
+HOMEPAGE="https://www.rust-lang.org/"
+COPYRIGHT="2018 The Rust Project Developers"
+LICENSE="MIT"
+REVISION="1"
+
+cargoVersion="0.28.0"
+rlsVersion="0.127.0"
+rustfmtVersion="0.6.1"
+srcGitRev2="7243155b1c3da0a980c868a87adebf00e0b33989"
+srcGitRev3="118e078c5badd520d18b92813fd88789c8d341ab"
+srcGitRev4="c5fb6dbe8154732b2af8367c75a9b079b2951154"
+srcGitRev5="748a5e6742db4a21c4c630a58087f818828e8a0a"
+srcGitRev7="134f419ee62714590b04712fe6072253bc2a7822"
+srcGitRev8="f51127530d46b9acbf4747c859da185e771cfcf3"
+srcGitRev9="d2ade31a52a417257742de72c5936a8a342a34b5"
+srcGitRev10="2a2f6d96c8dc578d2474742f14c9bab0b36b0408"
+srcGitRev11="dfd960b5f1a1751b22738fa34fd27b583f4618db"
+srcGitRev12="49279d715bc9bc979313e7c1056ada821ddc3ee2"
+srcGitRev13="f48fed70d4447445b586a35c4ae88683542ffc72"
+srcGitRev14="eebda16e4b45f2eed4310cf7b9872cc752278163"
+srcGitRev15="a19ca1cd91cf97777af8268a6136bd2e4648e189"
+srcGitRev16="1742229ebb7843a65c05ee495d8de5366fcc5567"
+SOURCE_URI="https://github.com/rust-lang/rust/archive/$portVersion.tar.gz"
+SOURCE_URI_2="https://github.com/rust-lang/llvm/archive/$srcGitRev2.tar.gz"
+SOURCE_URI_3="https://github.com/rust-lang/rust-installer/archive/$srcGitRev3.tar.gz"
+SOURCE_URI_4="https://github.com/rust-lang/libc/archive/$srcGitRev4.tar.gz"
+SOURCE_URI_5="https://github.com/rust-lang-nursery/nomicon/archive/$srcGitRev5.tar.gz"
+SOURCE_URI_6="https://github.com/rust-lang/cargo/archive/$cargoVersion.tar.gz"
+SOURCE_URI_7="https://github.com/rust-lang-nursery/reference/archive/$srcGitRev7.tar.gz"
+SOURCE_URI_8="https://github.com/rust-lang/book/archive/$srcGitRev8.tar.gz"
+SOURCE_URI_9="https://github.com/rust-lang-nursery/rls/archive/$srcGitRev9.tar.gz"
+SOURCE_URI_10="https://github.com/rust-lang-nursery/compiler-builtins/archive/$srcGitRev10.tar.gz"
+SOURCE_URI_11="https://github.com/rust-lang/compiler-rt/archive/$srcGitRev11.tar.gz"
+SOURCE_URI_12="https://github.com/rust-lang-nursery/rustfmt/archive/$srcGitRev12.tar.gz"
+SOURCE_URI_13="https://github.com/solson/miri/archive/$srcGitRev13.tar.gz"
+SOURCE_URI_14="https://github.com/rust-lang/rust-by-example/archive/$srcGitRev14.tar.gz"
+SOURCE_URI_15="https://github.com/rust-lang-nursery/stdsimd/archive/$srcGitRev15.tar.gz"
+SOURCE_URI_16="https://github.com/rust-lang-nursery/rust-clippy/archive/$srcGitRev16.tar.gz"
+SOURCE_FILENAME="rust-$portVersion.tar.gz"
+CHECKSUM_SHA256="0bbca54761be5302efc6e6fdb5f4dab4e0dec24458ca7c13538b7259885f9457"
```

Comment:
Suggestion:
```diff
 SOURCE_FILENAME="rust-$portVersion.tar.gz"
+SOURCE_FILENAME_2="rust-$portVersion-llvm-$srcGitRev2.tar.gz"
+SOURCE_FILENAME_3="rust-$portVersion-installer-$srcGitRev3.tar.gz"
+SOURCE_FILENAME_4="rust-$portVersion-libc-$srcGitRev4.tar.gz"
+SOURCE_FILENAME_5="rust-$portVersion-nursery-nomicon-$srcGitRev5.tar.gz"
+SOURCE_FILENAME_6="rust-$portVersion-cargo-$cargoVersion.tar.gz"
+SOURCE_FILENAME_7="rust-$portVersion-nursery-reference-$srcGitRev7.tar.gz"
+SOURCE_FILENAME_8="rust-$portVersion-book-$srcGitRev8.tar.gz"
+SOURCE_FILENAME_9="rust-$portVersion-nursery-rls-$srcGitRev9.tar.gz"
+SOURCE_FILENAME_10="rust-$portVersion-nursery-compiler-builtins-$srcGitRev10.tar.gz"
+SOURCE_FILENAME_11="rust-$portVersion-compiler-rt-$srcGitRev11.tar.gz"
+SOURCE_FILENAME_12="rust-$portVersion-nursery-rustfmt-$srcGitRev12.tar.gz"
+SOURCE_FILENAME_13="rust-$portVersion-miri-$srcGitRev13.tar.gz"
+SOURCE_FILENAME_14="rust-$portVersion-rust-by-example-$srcGitRev14.tar.gz"
+SOURCE_FILENAME_15="rust-$portVersion-nursery-stdsimd-$srcGitRev15.tar.gz"
+SOURCE_FILENAME_16="rust-$portVersion-rust-clippy-$srcGitRev16.tar.gz"
```
(Feel free to make changes to this suggestion, as e.g. `cargo-$cargoVersion.tar.gz` might be better than `rust-$portVersion-cargo-$cargoVersion.tar.gz`.)

Reply:
Is there any reason but aesthetics in the downloads-directory to make this change?

Reply:
Yep, more or less: [HaikuPorter-Guidelines#archive-name](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name)

But imo the best reason is that with a common prefix it makes it much easier to manually remove outdated tarballs.